frontend/package.json

```diff
@@ -40,8 +40,8 @@
     "start": "vite",
     "dev": "vite",
     "compile": "pnpm gen:sources && tsc --noEmit",
-    "gen:sources": "rimraf ./src/generated-sources && openapi-generator-cli generate",
-    "build": "rimraf ./src/generated-sources && openapi-generator-cli generate && tsc --noEmit && vite build",
+    "gen:sources": "rimraf ./src/generated-sources && cd ../contract-typespec/api && pnpm install && pnpm build && cd ../../frontend  && openapi-generator-cli generate",
```

Comment:
need to check that this works on windows™

@Nilumilak could you try out building this?